[Bug]: ocl-icd build failed

```
### Problem description

ake[1]: Entering directory '/home/builder/.termux-build/ocl-icd/build'
Making all in .
make[2]: Entering directory '/home/builder/.termux-build/ocl-icd/build'
aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ocl-icd/src  -Wall  -DOCL_ICD_PRINT_EXPECTED -I/data/data/com.termux/files/usr/include -Wno-deprecated-declarations  -fstack-protector-strong -Oz -c -o run_dummy_icd_through_our_ICDL-run_dummy_icd.o `test -f 'run_dummy_icd.c' || echo '/home/builder/.termux-build/ocl-icd/src/'`run_dummy_icd.c
aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ocl-icd/src  -Wall  -DOCL_ICD_PRINT_EXPECTED -I/data/data/com.termux/files/usr/include -Wno-deprecated-declarations  -fstack-protector-strong -Oz -c -o run_dummy_icd_through_our_ICDL-run_dummy_icd_gen.o `test -f 'run_dummy_icd_gen.c' || echo '/home/builder/.termux-build/ocl-icd/src/'`run_dummy_icd_gen.c
aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ocl-icd/src  -Wall  -DOCL_ICD_PRINT_EXPECTED -I/data/data/com.termux/files/usr/include -Wno-deprecated-declarations  -fstack-protector-strong -Oz -c -o run_dummy_icd_through_our_ICDL-run_dummy_icd_weak_gen.o `test -f 'run_dummy_icd_weak_gen.c' || echo '/home/builder/.termux-build/ocl-icd/src/'`run_dummy_icd_weak_gen.c
/bin/bash ./libtool  --tag=CC   --mode=compile aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ocl-icd/src  -Wall  -I/data/data/com.termux/files/usr/include -Wno-deprecated-declarations   -DETC_OPENCL_VENDORS=\"/data/data/com.termux/files/usr/etc/OpenCL/vendors\" -fstack-protector-strong -Oz -c -o libOpenCL_la-ocl_icd_loader.lo `test -f 'ocl_icd_loader.c' || echo '/home/builder/.termux-build/ocl-icd/src/'`ocl_icd_loader.c
libtool: compile:  aarch64-linux-android-clang -DHAVE_CONFIG_H -I. -I/home/builder/.termux-build/ocl-icd/src -Wall -I/data/data/com.termux/files/usr/include -Wno-deprecated-declarations -DETC_OPENCL_VENDORS=\"/data/data/com.termux/files/usr/etc/OpenCL/vendors\" -fstack-protector-strong -Oz -c /home/builder/.termux-build/ocl-icd/src/ocl_icd_loader.c  -fPIC -DPIC -o .libs/libOpenCL_la-ocl_icd_loader.o
/home/builder/.termux-build/ocl-icd/src/ocl_icd_loader.c:895:3: error: expected identifier
  CL_ICDL_OCL_VERSION=1,
  ^
/data/data/com.termux/files/usr/include/CL/cl_ext.h:3181:61: note: expanded from macro 'CL_ICDL_OCL_VERSION'
#define CL_ICDL_OCL_VERSION                                 1
                                                            ^
/home/builder/.termux-build/ocl-icd/src/ocl_icd_loader.c:896:3: error: expected identifier
  CL_ICDL_VERSION=2,
  ^
/data/data/com.termux/files/usr/include/CL/cl_ext.h:3182:61: note: expanded from macro 'CL_ICDL_VERSION'
#define CL_ICDL_VERSION                                     2
                                                            ^
/home/builder/.termux-build/ocl-icd/src/ocl_icd_loader.c:897:3: error: expected identifier
  CL_ICDL_NAME=3,
  ^
/data/data/com.termux/files/usr/include/CL/cl_ext.h:3183:61: note: expanded from macro 'CL_ICDL_NAME'
#define CL_ICDL_NAME                                        3
                                                            ^
/home/builder/.termux-build/ocl-icd/src/ocl_icd_loader.c:898:3: error: expected identifier
  CL_ICDL_VENDOR=4,
  ^
/data/data/com.termux/files/usr/include/CL/cl_ext.h:3184:61: note: expanded from macro 'CL_ICDL_VENDOR'
#define CL_ICDL_VENDOR                                      4
                                                            ^
/home/builder/.termux-build/ocl-icd/src/ocl_icd_loader.c:899:3: error: typedef redefinition with different types ('enum cl_icdl_info' vs 'cl_uint' (aka 'unsigned int'))
} cl_icdl_info;
  ^
/data/data/com.termux/files/usr/include/CL/cl_ext.h:3178:29: note: previous definition is here
typedef cl_uint             cl_icdl_info;
                            ^
/home/builder/.termux-build/ocl-icd/src/ocl_icd_loader.c:901:15: error: static declaration of 'clGetICDLoaderInfoOCLICD' follows non-static declaration
static cl_int clGetICDLoaderInfoOCLICD(
              ^
/data/data/com.termux/files/usr/include/CL/cl_ext.h:3197:1: note: previous declaration is here
clGetICDLoaderInfoOCLICD(
^
6 errors generated.
make[2]: *** [Makefile:846: libOpenCL_la-ocl_icd_loader.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/builder/.termux-build/ocl-icd/build'
make[1]: *** [Makefile:1046: all-recursive] Error 1
make[1]: Leaving directory '/home/builder/.termux-build/ocl-icd/build'
make: *** [Makefile:[614](https://github.com/johndebia2022/termux-packages/actions/runs/5063709277/jobs/9090605308#step:6:615): all] Error 2
Error: Process completed with exit code 2.


[logs_31.zip](https://github.com/termux/termux-packages/files/11549604/logs_31.zip)


### What steps will reproduce the bug?

build

### What is the expected behavior?

_No response_

### System information

termux-info:

```

```

```



[logs_31.zip](https://github.com/termux/termux-packages/files/11549608/logs_31.zip)

--- a/packages/ocl-icd/build.sh
+++ b/packages/ocl-icd/build.sh
@@ -3,21 +3,21 @@ TERMUX_PKG_DESCRIPTION="OpenCL ICD Loader"
 TERMUX_PKG_LICENSE="BSD 2-Clause"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=2.3.1
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_REVISION=3
 TERMUX_PKG_SRCURL=https://github.com/OCL-dev/ocl-icd/archive/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=a32b67c2d52ffbaf490be9fc18b46428ab807ab11eff7664d7ff75e06cfafd6d
 TERMUX_PKG_NO_STATICSPLIT=true
-TERMUX_PKG_BUILD_DEPENDS="opencl-headers"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
---enable-custom-vendordir=$TERMUX_PREFIX/etc/OpenCL/vendors
+--enable-custom-vendordir=${TERMUX_PREFIX}/etc/OpenCL/vendors
+--enable-official-khronos-headers
 "
-
-# https://www.khronos.org/registry/OpenCL/specs/2.2/html/OpenCL_ICD_Installation.html
-# Intepreting this as providing library "libOpenCL.so" with SONAME "libOpenCL.so" on Android
 
 termux_step_pre_configure() {
 	./bootstrap
 }
+
+# https://www.khronos.org/registry/OpenCL/specs/2.2/html/OpenCL_ICD_Installation.html
+# Intepreting this as providing library "libOpenCL.so" with SONAME "libOpenCL.so" on Android
 
 # https://github.com/termux/termux-packages/issues/7510
 # Removed handling of PREFIX/etc/OpenCL/vendors to match Desktop Linux ocl-icd behaviour

